### PR TITLE
fix: Log disk info of C drive

### PR DESCRIPTION
**Pull request by AbelHu**
Old code does not always work so fix it.

### CODE REVIEW

**Comment by AbelHu** [comment]
@mainred It works. You can see the disk info in the build logs.

File: vhdbuilder/packer/configure-windows-vhd.ps1 [hunk 1]
```diff
@@ -472,8 +482,7 @@ switch ($env:ProvisioningPhase) {
         Update-Registry -containerRuntime $containerRuntime
         Get-ContainerImages -containerRuntime $containerRuntime -windowsSKU $windowsSKU
         Get-FilesToCacheOnVHD -containerRuntime $containerRuntime
-        # Show disk space
-        Get-CimInstance -ClassName Win32_LogicalDisk
```
Comment (mainred) [inline vhdbuilder/packer/configure-windows-vhd.ps1:476]: In which case this does not work?

Comment (AbelHu) [reply vhdbuilder/packer/configure-windows-vhd.ps1:476]: No idea. I just found that it does not log the disk info as expected sometimes.

Comment (AbelHu) [reply vhdbuilder/packer/configure-windows-vhd.ps1:476]: @mainred do you have other concern for this PR and need me to update it?

**Review by mainred — APPROVED** [review]
Thanks for the fix.

**Pull request merged**